types/stake.go

```diff
@@ -76,6 +76,7 @@ type ValidatorSet interface {
 	ValidatorByConsAddr(Context, ConsAddress) Validator // get a particular validator by consensus address
 	ValidatorByVoteAddr(Context, []byte) Validator      // get a particular validator by vote address
 	TotalPower(Context) Dec                             // total power of the validator set
+	GetTotalVotingPower(ctx Context) Dec                // total voting power of the validator set
```

Comment:
You get `TotalPower` and `GetTotalVotingPower`, two quite similar functions.

The first one is to get from Pool only contains the bonded voting power, the second one is all voting power, including not bonded voting power, so we may give a different func name?

Reply:
how about `GetAnyStatusVotingPower`?